src/mod_path.rs

```diff
@@ -123,7 +124,7 @@ impl From<&ItemMod> for ModSegment {
     fn from(v: &ItemMod) -> Self {
         for attr in &v.attrs {
             if let Ok(Meta::NameValue(name_value)) = attr.parse_meta() {
-                if name_value.ident == "path" {
+                if name_value.path.to_token_stream().to_string() == "path" {
```

Comment:
What about using [`Path::is_ident`](https://docs.rs/syn/1.0.12/syn/struct.Path.html#method.is_ident) here? As I understand, it's made for this use case - and wouldn't require the non-dev dependency on `quote`.

```suggestion
                 if name_value.path.is_ident("path") { 
```